bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -417,3 +418,19 @@ def pythonize_rdataframe(klass):
                                       HistoProfileWrapper,
                                       model_class)
         setattr(klass, method_name, getter)
+
+
+@pythonization("RInterface<", ns="ROOT::RDF", is_prefix=True)
+def pythonize_rdataframeFilter(klass):
+    klass._OriginalFilter =  klass.Filter
+    klass.Filter = _PyFilter
+
+
+def _PyFilter(rdf, func, funcargs = {}, filter_name = ""):
```

Comment:
For the sake of keeping the same style, it would be better to place this helper function before `pythonize_rdataframe` (i.e. the pythonizor function is the last to appear in the file).